packages/replicache/src/kv/sqlite-store-test-util.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console */
+import sqlite3 from '@rocicorp/zero-sqlite3';
+import fs from 'fs';
+import path from 'path';
+import {SQLiteDatabaseManager} from './sqlite-store.ts';
+
+export const getTestSQLiteDatabaseManager = (logging: boolean = false) =>
+  new SQLiteDatabaseManager({
+    open: name => {
+      const filename = path.resolve(__dirname, `${name}.db`);
+      // this cannot be :memory: because multiple read connections must access
```

Comment:
it's technically doable via memory with some fancy options. But yeah, better to just keep it file based.